book/theme/pagetoc.css

```diff
@@ -0,0 +1,57 @@
+@media only screen and (max-width:1439px) {
```

Comment:
Since we're not customizing this and pagetoc.js, let mdbook-pagetoc generate them instead of adding them to version control.

Then you just need to add "theme" to book/.gitignore:

```
book
theme

node_modules
package-lock.json
src/super.wasm
super-example.bundle.js
super-example/wasm_exec.js
```